lib/factory_bot/linter.rb

```diff
@@ -1,9 +1,10 @@
 module FactoryBot
   class Linter
-    def initialize(factories, linting_strategy, factory_strategy = :create)
+    def initialize(factories, strategy: :create, traits: false, verbose: false)
```

Comment:
There's a fair amount of logic shift in this PR in addition to the verbose flag. Would it make sense to refactor this to options (sans-`verbose`) in one PR and then add the `verbose` flag as a separate PR?

Reply:
Yup, good point